apps/ios/SimpleXChat/API.swift

```diff
@@ -268,6 +276,13 @@ func decodeProperty<T: Decodable>(_ obj: Any, _ prop: NSString) -> T? {
     return nil
 }
 
+func json(_ obj: Any) -> String? {
+    if let d = try? JSONSerialization.data(withJSONObject: obj) {
+        return String(decoding: d, as: UTF8.self)
+    }
+    return nil
+}
```

Comment:
we already have some function that does encoding, why this is necessary?

Reply:
prettyJSON

Reply:
it's `prettyJSON` below, this one is regular, not pretty. it avoids unnecessary "\n"s which are packed into string anyway

Reply:
pass parameter :)